src/Uno.UWP/Devices/Sensors/SimpleOrientationSensor.Android.cs

```diff
@@ -1,31 +1,88 @@
 ﻿#if __ANDROID__
+using System;
 using Android.App;
 using Android.Content;
+using Android.Content.Res;
 using Android.Hardware;
 using Android.Runtime;
+using Android.Views;
+using Uno.UI;
+using Android.Provider;
+using static Android.Provider.Settings;
+using Android.Database;
+using Android.OS;
 
 namespace Windows.Devices.Sensors
 {
 	public partial class SimpleOrientationSensor : Java.Lang.Object, ISensorEventListener
 	{
+		#region Static
+
+		private static Orientation _defaultDeviceOrientation;
+
+		private static Orientation DefaultDeviceOrientation
+		{
+			get
+			{
+				if (_defaultDeviceOrientation == Orientation.Undefined)
+				{
+					var context = ContextHelper.Current;
```

Comment:
This may be null, can you add a meaningful check ?

Reply:
Yes sure, will add it. Thanks !